DataStreamerUWP/DataSourceManifest.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.DataStreamer.UWP
+{
+    public class DataSourceManifest
```

Comment:
each class should explain its purpose, context in which it will be used to make it more readable